src/workerd/server/server-test.c++

```diff
@@ -1138,6 +1138,185 @@ KJ_TEST("Server: override globalOutbound") {
   conn.recvHttp200("OK");
 }
 
+KJ_TEST("Server: connect() to default outbound") {
+  TestServer test(singleWorker(R"((
+    compatibilityDate = "2022-08-17",
```

Comment:
nit/question: is this intentionally using an older compat date?

Reply:
It's copy/pasted from another test. A compatibility date is required but for this test the date doesn't really matter.